### DISCUSSION TITLE
Configuring GitHub Actions without storing them in .github directory

### DISCUSSION THREAD

**Discussion by sryze** [community/community#8207] [Actions]
Please add the ability to configure actions without the need to create configuration files inside a repository. 

For example, AppVeyor has a web UI where you can do almost everything you can do in `appveyor.yml`.

**Comment by ethomson** [comment]
Hi @sryze, thanks for the feedback.  At the moment, we don't have plans to introduce a web UI for configuring GitHub Actions, but it's an idea that we should consider.

If we did this, it would update the YAML in `.github`.  At the moment all the github configuration, including for things like releases, issue templates, etc is in the `.github` folder.  Even if we added a web UI, we would keep the YAML the "source of truth" for your configuration.
